app/controllers/cart_controller.rb

```diff
@@ -1,12 +1,12 @@
 # frozen_string_literal: true
 
 class CartController < ApplicationController
-
   def index
     @orderables = Orderable.order(created_at: :desc)
-    if [current_user.profile&.user_id].present?
-      @order_histories = Orderable.where("customer_id LIKE ?", "%#{current_user.profile&.user_id}%").order(created_at: :desc)
-    end
+    return unless [current_user.profile&.user_id].present?
+
+    @order_histories = Orderable.where('customer_id LIKE ?',
+                                       "%#{current_user.profile&.user_id}%").order(created_at: :desc)
```

Comment:
current_user.id and current_user.profile&.user_id aren't same?

Reply:
done